### ISSUE TITLE
Notebook 7 surface view error

### ISSUE THREAD

**Issue by wangenau** [nglviewer/nglview#1071]
Yesterday with notebook 7.0.0 a new Jupyter version got released.
I tested nglview after updating, but I found that the surface view is broken after updating and displays nothing.
There was no error written to the terminal.

Here is a minimal example for testing with an example [CUBE file](https://github.com/nglviewer/nglview/files/12103774/CH4_density.zip)
```python
import nglview
view = nglview.NGLWidget()
view.add_component('CH4_density.cube')
view.add_surface()
view
```
The example works with the latest version 6 notebook 6.5.4.
Displaying molecules seems to work just fine.

**Comment by hainm** [comment]
thanks for your report, @wangenau

**Comment by hainm** [comment]
The issue is resolved in #1100 (available in `nglview 3.1.1`)

**Comment by wangenau** [comment]
Thanks a lot!
I tested it and it works as expected again.
